link/src/main/java/com/stripe/android/link/NativeLinkEnabled.kt

```diff
@@ -0,0 +1,6 @@
+package com.stripe.android.link
+
+object NativeLinkEnabled {
```

Comment:
Are there any security concerns with including this in the SDK? I think long-term we wouldn't want this setting to live client-side, because I think an app could theoretically access and manipulate this data. Are there similar concerns about exposing it now while testing? 

If we are ok with adding this, it should definitely not be public

Reply:
Yh, this should be restricted to library group, my bad. I'll update it.

Reply:
This should probably be a feature flag instead. We have the ability to not allow setting it to true when in release mode.

Reply:
This will be a flag in elementSession, but that's not ready yet. We'll also need a signal from the attestation module to tell us if hardware attestation is available. That's also not ready yet. I added this as a stop gap to manually enable native link. It will evolve into a proper gate when the pieces mentioned above are available. 

Is this still a risk if it's restricted to the library group? If yes, I can take it out and hardcode the launcher when I'm testing or creating a test build.

Reply:
You can use a local feature flag for now, which will prevent anything happening in published versions of the SDK. https://github.com/stripe/stripe-android/blob/f6131fdfae3f655b1bd8e4cc1c91b66d10adedea/stripe-core/src/main/java/com/stripe/android/core/utils/FeatureFlags.kt#L8

Reply:
Thanks!